xtokens/src/mock/tests.rs

```diff
@@ -0,0 +1,359 @@
+#![cfg(test)]
```

Comment:
Tests in this file look like for testing upstream deps, I don't see why we need them in `orml`.

Reply:
move tests here to xcm-support